### PR TITLE
Changes based on the feedback on clean up

### PR DESCRIPTION
**Pull request by IgorPidik**

### CODE REVIEW

File: lib/Document.php [hunk 1]
```diff
@@ -164,10 +168,9 @@ public function getPositionInImage()
      */
     public function loadImage(string $binaryJpeg)
     {
-        $tmp = tempnam('', 'verifai_image');
-        file_put_contents($tmp, $binaryJpeg);
-        $this->image = imagecreatefromjpeg($tmp);
-        unlink($tmp);
+        $this->image = imagecreatefromstring($binaryJpeg);
+        # cashed image is no longer valid
```
Comment (zout) [inline lib/Document.php]: You can leave this comment out

File: lib/Service.php [hunk 2]
```diff
@@ -38,7 +38,7 @@ class Service
      * Weather or not to check the SSL certificates while communicating
      * @var bool
      */
-    public $sslVerify = true;
+    public $sslVerify = 2;
```
Comment (zout) [inline lib/Service.php]: why a 2?

Comment (IgorPidik) [reply lib/Service.php]: `Notice: curl_setopt(): CURLOPT_SSL_VERIFYHOST no longer accepts the value 1, value 2 will be used instead in ...`
Not sure if it's only certain php version issue or general thing

File: lib/Document.php (hunk 1 above)
Comment (IgorPidik) [reply lib/Document.php]: check

File: lib/Service.php (hunk 2 above)
Comment (zout) [reply lib/Service.php]: Then i'd translate it to a `2` when you pass it to Curl. We shouldn't be lib vendor dependant on this. If we switch to `file_get_contents` for example, we can't pass a `2`.

Comment (IgorPidik) [reply lib/Service.php]: check, should be fine now

**Pull request merged**